### ISSUE TITLE
[BUG] Playing status does not show up to other users

### ISSUE THREAD

**Issue by FrezingCrazy** [hypherionmc/simple-rpc-public#93]
**Describe the bug**
Edited config file for my modpack, added to my mod list, running Discord, and I do see my profile playing minecraft modpack with text I written. But no one else does, the same if somebody else would turn game with mod and config on, they would see themselves in launched game, but not me or anyone else.

**To Reproduce**
None

**Expected behavior**
For all users who click on profile see playing game with my modpack actually, instead of seeing nothing as just online in app.

**Versions (DO NOT USE LATEST! BE SPECIFIC!):**
 - OS: Windows 10
 - JAVA Version: Java 8 Update 361
 - Mod Version: SimpleRPC 3.1.1-1.12.2
 - Minecraft Version: ForgeOptifine 1.12.2

**Comment by hypherionmc** [comment]
The only thing I know of that causes this, is discord not liking certain characters in "non english" languages.

We've only ever had one case of this, and it was, if I remember correctly, a single chinese character that broke it. It's not a bug in the mod, but an issue on discords side. 


My normal recommendation for testing/figuring this out, is start with a clean default config (and see if that works), the slowly start adding your modifications back until it breaks again

**Comment by FrezingCrazy** [comment]
Yeah, I'm using Russian Cyrillic and sentences are written on it.

**Comment by FrezingCrazy** [comment]
I found out that reason for that, is when you try to add buttons that has not Latin letters
